Modules/Example/CMakeLists.txt

```diff
@@ -25,16 +25,10 @@ install(TARGETS QcExample
         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 
-# Install the config files for the analysis.
-# The command "install" cannot be used because it would be executed after o2_add_qc_workflow which relies on
-# the config files being already installed in the right place.
-configure_file("etc/analysisDirect.json" "${CMAKE_INSTALL_PREFIX}/etc/analysisDirect.json")
-configure_file("etc/analysisDerived.json" "${CMAKE_INSTALL_PREFIX}/etc/analysisDerived.json")
-
 # ---- Workflows for analysis ----
 
-o2_add_qc_workflow(WORKFLOW_NAME o2-qc-example-analysis-direct CONFIG_FILE_PATH ${CMAKE_INSTALL_PREFIX}/etc/analysisDirect.json)
-o2_add_qc_workflow(WORKFLOW_NAME o2-qc-example-analysis-derived CONFIG_FILE_PATH ${CMAKE_INSTALL_PREFIX}/etc/analysisDerived.json)
+o2_add_qc_workflow(WORKFLOW_NAME o2-qc-example-analysis-direct CONFIG_FILE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/etc/analysisDirect.json)
+o2_add_qc_workflow(WORKFLOW_NAME o2-qc-example-analysis-derived CONFIG_FILE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/etc/analysisDerived.json)
```

Comment:
The DPL dumps should point to the configuration files in the installation paths, not the ones in source dir.

Reply:
ah right, that was the thing. It needs to point to the correct path to generate a proper json. Fixing